.gitignore

```diff
@@ -34,3 +34,4 @@ Carthage
 # `pod install` in .travis.yml
 #
 # Pods/
+.swiftpm
```

Comment:
Doesn't `.build` need to be in here to avoid people checking in local builds?

Reply:
You’re more likely the build it using spm from outside this repo but I guess it doesn’t hurt. I’ll add it tomorrow morning 👌